docs/core/dlc.md

```diff
@@ -0,0 +1,57 @@
+---
+id: dlc
+title: Discreet Log Contract Data Structures
+---
+
+Bitcoin-S now has support for the basic data structures and algorithms involved in the setup and execution of Discreet Log Contracts (DLCs) in the package `org.bitcoins.core.protocol.dlc` as well as basic TLV format and LN Message format support at `org.bitcoins.core.protocol.tlv` which can be useful for many off-chain protocols including DLCs and Lightning.
+
+Please note that this code is experimental as the DLC specification is a work-in-progress and so this code is not stable and is subject to change. This means that the DLC code in the official release may be out-of-date and may conflict with other implementations' results.
+
+> If you require the most up-to-date DLC data structures and algorithms, you should use the master branch.
+> If you are interested in actually setting up and executing DLCs, you should checkout the [adaptor-dlc branch](https://github.com/bitcoin-s/bitcoin-s/tree/adaptor-dlc).
+
+Let's now cover the main data structures and interfaces supported.
+
+## DLCPayoutCurve
+
+[DLCPayoutCurve.scala] provides an interface for serializing and evaluating payout curves for DLCs as specified in the [Payout Curve Specification](https://github.com/discreetlogcontracts/dlcspecs/blob/c4fb12d95a4255eabb873611437d05b740bbeccc/PayoutCurve.md). This file supports arbitrary polynomial interpolation.
```

Comment:
This is done a bunch with no following `(link)`, is tha intentional?

Reply:
It's in the PR description that he is waiting for the code PR to be merged.